### ISSUE TITLE
BUG - DOCKER_TLS env var is ignored

### ISSUE THREAD

**Issue by WTFKr0** [docker/app#313]
**Description**

Can't use DOCKER_TLS env var with docker-app

**Steps to reproduce the issue:**
1. Set DOCKER_HOST env var to a TLS remote socket (tcp://mydocker:2376)
2. Set TLS_VERIFY env var to 1

**Describe the results you received:**
```
Error: Get http://mydocker:2376/v1.37/info: net/http: HTTP/1.x transport connection broken: malformed HTTP response "\x15\x03\x01\x00\x02\x02".
* Are you trying to connect to a TLS-enabled daemon without TLS?
```

**Describe the results you expected:**
docker-app deploy my app

**Output of `docker version`:**

```
Client:
 Version:      18.03.1-ce
 API version:  1.37
 Go version:   go1.9.4
 Git commit:   9ee9f40
 Built:        Thu Apr 26 04:27:49 2018
 OS/Arch:      linux/amd64
 Experimental: false
 Orchestrator: swarm

Server:
 Engine:
  Version:      18.03.1-ce
  API version:  1.37 (minimum version 1.12)
  Go version:   go1.9.4
  Git commit:   9ee9f40
  Built:        Thu Apr 26 04:27:49 2018
  OS/Arch:      linux/amd64
  Experimental: false
```

**Output of `docker-app version`:**

```
Version:      v0.3.0
Git commit:   fba6a09
Built:        Fri Jun 29 13:09:30 2018
OS/Arch:      linux/amd64
Experimental: off
Renderers:    none
```

**Comment by vdemeester** [comment]
@WTFKr0 thanks for the report. I think https://github.com/docker/app/pull/302 will fix it, so let's see if the next releases fixes that for you :wink: (0.4.0 should be in a few days)

**Comment by WTFKr0** [comment]
Thanx

As a workaround I can : 
```
docker-app render myimage:mytag | docker stack deploy -c - stack_name
```

**Comment by chris-crone** [comment]
Fixed by #302.

@WTFKr0 We're planning on doing a release soon. Please reopen this issue if you encounter it with that.
